yarn.lock

```diff
@@ -14831,6 +14831,13 @@ ovh-api-services@^11.0.0:
   dependencies:
     lodash "^4.17.20"
 
+ovh-api-services@^10.1.0:
```

Comment:
Already updated to v11.0.0 on `develop` branch (see #3869).

In order to be able to merge this Pull Request I will rebase it on `develop` branch.

Reply:
Rebased successfully on `develop` branch.